subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/client/kubernetes/ReservationDeploymentFactory.kt

```diff
@@ -214,17 +220,17 @@ class ReservationDeploymentFactory(
         device: Device
     ): Map<String, String> {
         return mapOf(
-            "type" to buildType, // teamcity or local
+
+            /**
+             * used to distinguish different strategies for clearing leaked kubernetes deployments
+             * for incorrectly finished builds
+             * see [com.avito.ci.DeploymentEnvironment]
+             */
+            "type" to buildType,
```

Comment:
It'd good to get this value from the code. Otherwise, it's easy to break this contract in `DeploymentEnvironment`

Reply:
that's right, but this modules not linked; and introducting new "bridge" module only for pair of constants seems overkill for me;

long term support will not be an issue because of new service to manager deployments